app/assets/stylesheets/home.scss

```diff
@@ -1,3 +1,6 @@
 // Place all the styles related to the home controller here.
 // They will automatically be included in application.css.
 // You can use Sass (SCSS) here: http://sass-lang.com/
+.home-image-margin-bottom {
+  margin-bottom: 5%;
```

Comment:
I believe there is a more clean way to do that. Check this out:

https://v4-alpha.getbootstrap.com/utilities/spacing/

You can have a similar result with `mb-2`. Also, using percentage for margin may create unexpected results in the layout since the space will vary with screen height.